jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/common/TextFieldTests.java

```diff
@@ -21,13 +21,16 @@
 
 public class TextFieldTests extends TestsInit {
 
+    private String text = "TextField";
+    private final String symbols = "`!@#";//$%ˆ*()_+˜@[\"№;:?-=]{}'/\\|<>";
+
+
```

Comment:
Extra line should be removed

Reply:
done:

public class TextFieldTests extends TestsInit {

    private String text = "TextField";
    private final String symbols = "`!@#"; //$%ˆ*()_+˜@[\"№;:?-=]{}'/\\|<>";

    @BeforeMethod
    public void before() {
        shouldBeLoggedIn();
        html5Page.shouldBeOpened();
        name.setText(text);
    }